src/main/java/org/broadinstitute/hellbender/engine/spark/datasources/ReadsSparkSink.java

```diff
@@ -125,9 +167,9 @@ public static void writeReads(
 
         if (format == ReadsWriteFormat.SINGLE) {
             header.setSortOrder(SAMFileHeader.SortOrder.coordinate);
-            writeReadsSingle(ctx, absoluteOutputFile, samReads, header, numReducers);
+            writeReadsSingle(ctx, absoluteOutputFile, absoluteReferenceFile, samOutputFormat, samReads, header, numReducers, conf);
         } else if (format == ReadsWriteFormat.SHARDED) {
-            saveAsShardedHadoopFiles(ctx, absoluteOutputFile, samReads, header, true);
+            saveAsShardedHadoopFiles(ctx, absoluteOutputFile, absoluteReferenceFile, samOutputFormat, samReads, header, true, conf);
```

Comment:
Does `conf` need to be a separate parameter to these methods, given that it can be derived from `ctx`?